File: WebApplication/WebApplication/Models/TextProcessing/AnalysedTextModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebApplication.Models.TextProcessing
{
    public class AnalysedTextModel
    {
        public string Text { get; set; }
        public List<TermModel> Terms { get; set; }
        public List<AnalysedParagraphModel> Paragraphs { get; set; }

        public decimal SemanticSize { get; set; }
    }
}

File: WebApplication/WebApplication/ClientApp/src/models/text-analysis/analysed-text.model.ts
import { TermModel } from "../terms/term.model";
import { AnalysedParagraphModel } from "./analysed-paragraph.model";

export class AnalysedTextModel {
  text: string;
  terms: TermModel[];
  paragraphs: AnalysedParagraphModel[];
  semanticSize: number;
}
